cat.md

```diff
@@ -40,9 +46,30 @@ Cat Semantics
 
     syntax CatStep ::= "bite" String Address
  // ----------------------------------------
+    rule <k> Cat . bite ILK URN
+          => call Vat . grab ILK URN THIS VOWADDR (-1 *Rat minRat(INK,LUMP)) (-1 *Rat minRat(ART, minRat(INK,LUMP) *Rat ART /Rat INK))
+          ~> call Vow . fess (ART *Rat RATE)
+          ~> call Flip FLIP . kick URN VOWADDR (ART *Rat RATE *Rat CHOP) minRat(INK,LUMP) 0
+         ...
+         </k>
+         <this> THIS </this>
+         <cat-live> true </cat-live>
+         <cat-ilks>...
+           ID |-> Ilk(... flip: FLIP, chop: CHOP, lump: LUMP)
+         ...</cat-ilks>
+         <vat-ilks>...
+           ID |-> Ilk(... rate: RATE, spot: SPOT)
+         ...</vat-ilks>
+         <vat-urns>...
+           ID |-> Urn( INK, ART )
+         ...</vat-urns>
+         <vow-addr> VOWADDR </vow-addr>
```

Comment:
Looks like `vow` is a `file`able field, so we should have a cell `<cat-vow>` which points to the correct `VOWADDR` to use: https://github.com/makerdao/dss/blob/master/src/cat.sol#L101